lib/sprockets/source_map_comment_processor.rb

```diff
@@ -21,7 +24,9 @@ def self.call(input)
       uri, _ = env.resolve!(input[:filename], accept: map_type)
       map = env.load(uri)
 
-      path = PathUtils.relative_path_from(input[:filename], map.full_digest_path)
+      uri, _ = URIUtils.parse_asset_uri(input[:uri])
+      uri = env.expand_from_root(_[:index_alias]) if _[:index_alias]
```

Comment:
what is `_[]` doing? Are you using the `_` variable? If we should name it, usually `_` indicates that the variable isn't used.